index.html

```diff
@@ -1280,6 +1270,158 @@ <h3>
           conventions or limitations of the host operating system.
         </p>
       </section>
+      <section>
+        <h3>
+          `*_localized` members
+        </h3>
+        <p>
+          A <dfn>localizable member</dfn> is a [=manifest=] member that can be
+          localized. Each [=localizable member=] of the [=manifest=] has a
+          corresponding <dfn>localized member</dfn> with the suffix
+          `_localized`. For example, the [=manifest/name=] member is a
+          [=localizable member=], where `name_localized` is the corresponding
+          [=localized member=].
+        </p>
+        <p>
+          [=Localized members=] contain a [=localization map=] that defines
+          [=localized values=] for different locales denoted by a [=language
+          tag=]. If present and if the [=localization map=] contains a
+          [=localized value=] that matches the user's localization settings,
+          the user agent SHOULD override the value of the [=localizable
+          member=].
```

Comment:
I would probably explain this differently? For one thing, I think it is not helpful to talk about the localizations "overriding" the value. It's not really an override: it **_is_** the value. The `name` (etc.) is the _default_, used when no applicable localized value is available. The default has a locale associated with it (represented by the `lang` member), but the application does not do language matching against this value. 

Perhaps something like:

```suggestion
A [=localized member=] contains a [=localization map=] that provides
[=localized values=] for the given [=localizable member=] in the application.
Each [=localized value=] is identified by a [=language tag=].
The user agent SHOULD use the user's localization settings to select
the [=localized value=] whose language tag best matches the user's preference.
When no such [=localized value=] is available, the default representation is used.
```